src/annotations/actions/thunks.ts

```diff
@@ -0,0 +1,15 @@
+import {getAnnotations} from 'src/annotations/api'
+import {Dispatch} from 'react'
+
+import {
+  setAnnotations,
+  Action as AnnotationAction,
+} from 'src/annotations/actions/creators'
+
+export const fetchAndSetAnnotations = () => async (
```

Comment:
Love the naming here. I dislike it when fetch operations have a fetch<thing> call and then a set<thing> action... gets confusing to look at before coffee.